src/main/scala/io/viash/config/ArgumentGroup.scala

```diff
@@ -25,6 +25,18 @@ case class ArgumentGroup(
   @description("The name of the argument group.")
   name: String,
 
+  @description("A clean notation of the argument group's name. This is used when generating documentation.")
+  @example("label: an argument group", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  label: Option[String] = None,
+
+  @description("A short summary of the argument group. This is used when generating documentation.")
+  @example("summary: This argument group relates to function X", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  summary: Option[String] = None,
+
   @description("Description of foo`, a description of the argument group. Multiline descriptions are supported.")
   description: Option[String] = None,
```

Comment:
I noticed something wrong with the `@description` of `description`, and in the meantime I made minor tweaks to the other `@descriptions` as well.

```suggestion
  @description("A clean version of the argument group's name. This is only used for documentation.")
  @example("label: \"My argument group\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  label: Option[String] = None,

  @description("A one-sentence summary of the argument group. This is only used for documentation.")
  @example("summary: \"Arguments related to functionality XYZ\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  summary: Option[String] = None,

  @description("A description of the argument group. This is only used for documentation. Multiline descriptions are supported.")
  @example(
    """description: |
      |  A (multiline) description of the purpose of the arguments
      |  in this argument group.""".stripMargin, "yaml")
  @default("Empty")
  description: Option[String] = None,